src/libcore/mem.rs

```diff
@@ -417,14 +511,25 @@ pub fn replace<T>(dest: &mut T, mut src: T) -> T {
 
 /// Disposes of a value.
 ///
-/// While this does call the argument's implementation of `Drop`, it will not
-/// release any borrows, as borrows are based on lexical scope.
+/// While this does call the argument's implementation of [`Drop`][drop],
+/// it will not release any borrows, as borrows are based on lexical scope.
 ///
 /// This effectively does nothing for
 /// [types which implement `Copy`](../../book/ownership.html#copy-types),
 /// e.g. integers. Such values are copied and _then_ moved into the function,
 /// so the value persists after this function call.
 ///
+/// This function is not magic; it is literally defined as
+///
+/// ```rust,ignore
```

Comment:
It's actually runnable; I was thinking the name clash with the prelude would be a problem, but it just shadows I think.